samples/VanillaJSTestApp/JavascriptSPA/authConfig.js

```diff
@@ -2,12 +2,22 @@
 const msalConfig = {
     auth: {
         clientId: "4b0db8c2-9f26-4417-8bde-3f0e3656f8e0",
+        redirectUri: "http://localhost:30662/"
     },
     cache: {
         cacheLocation: "localStorage", // This configures where your cache will be stored
         storeAuthStateInCookie: false, // Set this to "true" if you are having issues on IE11 or Edge
+    },
+    system: {
+        telemetry: {
+            applicationName: 'msalVanillaTestApp',
+            applicationVersion: 'test1.0',
+            telemetryEmitter: (events) => {
+                console.log(events);
```

Comment:
I would doing something like `console.log("Telemetry Event: ", event)` to differentiate from other logs.

Reply:
addiong